float64[6] act_pos
int32[6] STATUS
